src/run.ts

```diff
@@ -59,7 +59,13 @@ export async function run() {
         namespace = 'default';
     }
     let action = core.getInput('action');
-    let manifests = manifestsInput.split('\n');
+    let manifests = manifestsInput.split(/[\n,]+/);
+
+    if (manifests.length > 0) {

```

Comment:
@shigupt202 I had to add this check explicitly because this looks like a bug. When we provide manifest names with a whitespace it gets added to the file path. Have we heard such cases?

Reply:
I'm not sure if anyone got into this issue before. But this looks like a valid change.